tests/envs/test_registration.py

```diff
@@ -89,7 +110,7 @@ def test_missing_lookup():
 def test_malformed_lookup():
     registry = registration.EnvRegistry()
     try:
-        registry.spec(u"“Breakout-v0”")
+        registry.spec("“Breakout-v0”")
```

Comment:
Could you revert this change? We can include this in a different PR where we clean up these issues.

Reply:
The thing is, if I put back the `u` before the string, the `black` pipeline fails, so I'd leave it this way.